Week-2/Advanced HTML/Advanced/review.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Restaurant Reviews - Add review</title>
+  </head>
+  <body>
+    <header>
+      <h1>Restaurant Reviews</h1>
+      <ul>
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="./review.html">Add Review</a></li>
+        <li><a href="./about.html">About</a></li>
+      </ul>
+    </header>
+    <section>
+      <h3>Add Review</h3>
+      <form>
+        <div>
+          <label for="name">Name:</label>
+          <input type="text" id="name" />
```

Comment:
A good practice of adding input HTML tags in the document is putting the input field and the label inside a label tag. So this HTML code can look like this

`<label>
<span>Name:</span>
<input type="text" id="name" />
</label>`

This has a lot of accessibility advantage as well as it removes the need for the name attribute in the label html tag.